### ISSUE TITLE
Failing case if extra directory is added

### ISSUE THREAD

**Issue by chinyeungli** [nexB/deltacode#17]
I did some simple tests and here is my finding:

I use "balloontip-1.1.1.jar" as a sample file.

1. Created 2 directories `d1/` and `d2/` and put the test file in it and then compare these 2 directories. The output is unchanged which is correct.

2. Same setup as (1) but create a new subdirectory named `test/` under `d1/` and put  balloontip-1.1.1.jar in it.
Both the 
`d1/balloontip-1.1.1.jar`
`d1/test/balloontip-1.1.1.jar`
are returned as added.

and the `d2/balloontip-1.1.1.jar` is returned as `removed`

which is not correct as the `d1/balloontip-1.1.1.jar` and `d2/balloontip-1.1.1.jar` should return unchanged while the `d1/test/balloontip-1.1.1.jar` is consider as added.

3. Same setup as (1) but create a new `root/` directory and put the `d1/` in it and run the deltacode from `root/` to `d2/`. The output is unchanged which is correct.

**Comment by pratik0316** [comment]
@chinyeungli @johnmhoran @majurg how about we ignore the align scans for now, after the integration of VirtualCodebse?
Maybe we could follow up in a separate branch apart from the main branch.

Let's consider a directory structure as :
![New Directory](https://user-images.githubusercontent.com/44219625/114314641-15f81680-9b19-11eb-9849-43cb308f6642.png)
 `New directory`


![Old Directory](https://user-images.githubusercontent.com/44219625/114314660-28725000-9b19-11eb-8bbd-f831cbc588ed.png)
`Old Directory`

Now if the `a1.py` is having the same `sha1` we are treating them as unchanged files, we completely ignore that their main directories are different. Mainly owing to (`alignscans / fix trees`).

But instead of that what if we do not allow changing the main root directory?
Now what I propose is we should also treat them as per their main root directory (not just their `sub dir`).
If we do so we do not need the extra burden of aligning the scans .

The scans will be aligned as they are loaded from the `Virtual Codebase` as a `resource` objects.
We can safely ignore all aligning.

Now for a file to have the status as `unmodified` it must have the same path(full path) along with the same sha1.
for a file of status moved it must exist in some other subdirs in `new_scan` along with that it must have the same `sha1`.
And so on for other status ....  

And also the codebase would be a lot cleaner than now

@chinyeungli @johnmhoran @majurg need your views upon this

**Comment by steven-esser** [comment]
@Pratikrocks I agree with removing/ignoring alignment for the first implementation of adding virtualcodebase.

**Comment by pratik0316** [comment]
Okay
